### ISSUE TITLE
--specieslist TypeError: stat: path should be string ...

### ISSUE THREAD

**Issue by rpetit3** [aldertzomer/RFPlasmid#11]
I'm using the pip install version.

```
rfplasmid --specieslist


Traceback (most recent call last):
  File "/local/home/rpetit/miniconda3/envs/diamond/lib/python3.9/site-packages/RFPlasmid/rfplasmid.py", line 41, in <module>
    if os.path.isdir(input_directory):
  File "/local/home/rpetit/miniconda3/envs/diamond/lib/python3.9/genericpath.py", line 42, in isdir
    st = os.stat(s)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```

```
python --version
Python 3.9.0

uname -a
Linux staphopia 4.19.0-12-amd64 #1 SMP Debian 4.19.152-1 (2020-10-18) x86_64 GNU/Linux
```

**Comment by aldertzomer** [comment]
@LindavdGvB  can you check?

**Comment by rpetit3** [comment]
This is fixed as of v0.0.12, going to close. Thank you!

```
python3 /local/home/rpetit/miniconda3/envs/rfplasmid/lib/python3.7/site-packages/RFPlasmid/rfplasmid.py --specieslist
Available species models:
Bacillus
Borrelia
Burkholderia
Campylobacter
Clostridium
Corynebacterium
Cyanothece
Enterococcus
Francisella
Lactobacillus
Lactococcus
Legionella
Listeria
Pseudomonas
Rhizobium
Staphylococcus
Streptomyces
Vibrio
Xanthomonas
Enterobacteriaceae
Bacteria

```
